src/app/views/menu.coffee

```diff
@@ -26,6 +25,7 @@ module.exports = class Menu extends BaseView
     backup: ->
         app.layout.closeMenu()
         app.init.launchBackup()
+        app.router.forceRefresh()
```

Comment:
Fix synchronization date in configuration page when I click on synchronization button